### DISCUSSION TITLE
How to specify "Ctrl-m" for "move"?

### DISCUSSION THREAD

**Discussion by hopezh** [zellij-org/zellij#2499] [Q&A]
I used the following settings in the `config.kdl` file, but "Ctrl-m" will not activate the "move" mode. Instead, `Ctrl-h` is still activating the move mode. Only the hint display is changed to `<m> MOVE`. 

So, how to change the settings to use `<C-m>` to activate the `MOVE` mode? 

```
    move {
        bind "Ctrl m" { SwitchToMode "Normal"; }
        bind "n" "Tab" { MovePane; }
        bind "p" { MovePaneBackwards; }
        bind "h" "Left" { MovePane "Left"; }
        bind "j" "Down" { MovePane "Down"; }
        bind "k" "Up" { MovePane "Up"; }
        bind "l" "Right" { MovePane "Right"; }
    }
```

```
    shared_except "move" "locked" {
        bind "Ctrl m" { SwitchToMode "Move"; }
    }
```

**Comment by Lockszmith-GH** [comment]
`Ctrl`+`M` is `<Enter>`, so mapping this would probably not be possible.

See https://unix.stackexchange.com/questions/283271/question-about-behavior-of-control-key-shortcuts
And https://en.wikipedia.org/wiki/C0_and_C1_control_codes#CR

**Comment by d0u9** [comment]
Same situation. I want to totally unbind `ctrl-h` and use `ctrl-u` or whatever other configurable keys for move mode.

I've tried:

```
keybinds {
    normal {
        bind "Ctrl u" { SwitchToMode "Move"; }
    }
    move {
        bind "Ctrl u" { SwitchToMode "Normal"; }
    }
    shared_except "locked" {
        bind "Ctrl u" { SwitchToMode "Move"; }
    }
        shared_except "move" "locked" {
        bind "Ctrl u" { SwitchToMode "Move"; }
    }
```

Even I configured zellij as above, `ctrl-h` key is still not freed, and can be used to switch in/out move mode.

**Reply by tlinford** [reply]
I think you need to clear the defaults altogether, or unbind ctrl-h: https://zellij.dev/documentation/keybindings-binding.html#overriding-keys

**Reply by d0u9** [reply]
Yeah, need to clear `ctrl-h`. Thank you very much.
